packages/web/components/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import Logo from './Logo';
+import NavigationButtons from './NavigationButtons';
+
+interface IProps {
+  height: number;
+}
+
+const Header: React.FC<IProps> = React.memo(({ height }) => {
+  const styles = {
+    backgroundColor: '#26abff',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    height,
+    padding: 20
+  };
+  return (
+    <div css={styles}>
+      <Logo title="InstaClone" css={{ color: '#fff' }} />
+      <NavigationButtons />
+    </div>
+  );
+});
+
+Header.displayName = 'Header';
+
+export default Header;
```

Comment:
Eksportuj wszystkie komponenty jako nie `default` ! Przy większych komponentach może nam się zdarzyć, że z jednego pliku będziemy chcieli eksportować dwie zmienne, czy to będzie kontekst, czy interfejs, czy cokolwiek... Nawet jeśli eskportuejsz jedną zmienną (tak jak w tym przypadku), to warto przyjąć jeden styl w projekcie i się go trzymać. Poza tym importowanie z wąsami `{}` pozwala nam na zmianę nazwy zaimportowanego komponentu poprzez `as`. I nie mówię już mamy jedną linijke kodu mniej - `export const Header: React.FC<IProps> = ... `

Reply:
To tyczy się też wszystich poniższych komponentów.